ReactWindows/ReactNative.Tests/UIManager/UIManagerModuleTests.cs

```diff
@@ -5,37 +5,38 @@
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Windows.UI.ViewManagement;
 
 namespace ReactNative.Tests.UIManager
 {
     [TestClass]
     public class UIManagerModuleTests
     {
-        [TestMethod]
+        [TestMethod, STAThread]
```

Comment:
>STAThread [](start = 21, length = 9)

I'd rather not use this attribute until I understand it more.  It shouldn't be needed if you take the suggestion in UIManager to only parameterize for .NET 4.6 (since we can get the application view at any time from UWP).